schema/core/primitive/sluggified_entry.json

```diff
@@ -0,0 +1,21 @@
+{
+    "schemaId": "core-primitive-sluggified-entry",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "sluggified entry",
```

Comment:
There should be one "g" - slugified - see https://hexdocs.pm/slugify/Slug.html, for instance